pyrightconfig.json

```diff
@@ -9,35 +9,24 @@
         // test cases use a custom config file
         "stubs/**/@tests/test_cases"
     ],
-    "typeCheckingMode": "basic",
-    "strictListInference": true,
-    "strictDictionaryInference": true,
-    "strictSetInference": true,
-    "reportFunctionMemberAccess": "error",
-    "reportMissingTypeStubs": "error",
-    "reportUnusedImport": "error",
-    "reportUnusedClass": "error",
-    "reportUnusedFunction": "error",
-    "reportUnusedVariable": "error",
-    "reportDuplicateImport": "error",
-    "reportUntypedFunctionDecorator": "error",
-    "reportUntypedClassDecorator": "error",
-    "reportUntypedBaseClass": "error",
-    "reportUntypedNamedTuple": "error",
-    "reportConstantRedefinition": "error",
-    "reportInvalidStringEscapeSequence": "error",
-    "reportUnknownArgumentType": "error",
-    "reportUnknownLambdaType": "error",
-    "reportMissingTypeArgument": "error",
-    "reportInvalidStubStatement": "error",
-    "reportInvalidTypeVarUse": "error",
-    "reportUnsupportedDunderAll": "error",
-    "reportInconsistentConstructor": "error",
-    "reportTypeCommentUsage": "error",
-    "reportUnnecessaryComparison": "error",
+    "typeCheckingMode": "strict",
+    // Allowed in base settings for incomplete stubs, checked in stricter settings
+    "reportIncompleteStub": "none",
+    "reportMissingParameterType": "none",
+    "reportUnknownMemberType": "none",
+    "reportUnknownParameterType": "none",
+    "reportUnknownVariableType": "none",
+    // Extra strict settings
+    "reportCallInDefaultInitializer": "error",
+    "reportImplicitStringConcatenation": "error",
+    "reportUninitializedInstanceVariable": "error",
```

Comment:
This one won't have any effect for stubs, right? (Same comment for the same line in the stricter config file)

Reply:
Indeed, just like `reportMissingSuperCall`. I've updated it.